src/main/resources/application.yml

```diff
@@ -19,6 +19,7 @@ consentmanager:
     userCreationTokenValidity: ${USER_CREATION_TOKEN_VALIDITY}
     userIdSuffix: ${USER_ID_SUFFIX}
   otpservice:
+    expirationTime: 300
```

Comment:
This is not a independent value. In the sense, this value has to reflect the otp service's expiration time. So it won't make sense for this to be changing based on different environments.

Reply:
Thought It's not different for environments. Doesn’t it come from ENv varibles. Because in future if we have to change the time then we can directly change in ENV variable right?

Reply:
IMO that will be cluttering and needless indirection.

Reply:
See, the thing is If It's coming from EVN varibles then you don't need to do the code changes but the you need to change the Env file.

Reply:
Let me update it then

Reply:
Updated